src/psyclone/dynamo0p3.py

```diff
@@ -4216,7 +4588,7 @@ def descriptor(self):
     def ref_name(self, function_space=None):
         ''' Returns the name used to dereference this type of argument. '''
         if not function_space:
-            if self._type in VALID_OPERATOR_NAMES:
+            if self.is_operator:
```

Comment:
document args in parent ref_name method docstring.

Reply:
Done.